ui/app/AppLayouts/Chat/helpers/CommunityPermissionsHelpers.qml

```diff
@@ -3,37 +3,42 @@ pragma Singleton
 import QtQml 2.14
 
 QtObject {
-    readonly property QtObject _d: QtObject {
-        id: d
 
-        function getByKey(model, key) {
+    function getTokenByKey(model, key) {
+        if(model) {
             for (let i = 0; i < model.count; i++) {
                 const item = model.get(i)
-                if (item.key === key)
+                if (item.key === key) {
                     return item
+                }
+                else if(item.subItems) {
```

Comment:
No `else` after return

Reply:
I don't understand it. Could you give me more details, please?! :pray: :sweat_smile:

Reply:
```qml
if (item.key === key) 
  return item

if (item.subItems) {
  const subitem = getTokenByKey(item.subItems, key)
  if (subitem !== null)
    return subitem
}
```